dev/src/components/CourseUtilityLinks.tsx

```diff
@@ -13,7 +13,7 @@ const CourseUtilityLinks = (): JSX.Element => {
     const repo = state.repositories[courseID]
 
     return (
-        <div className="col-lg-3" >
+        <div className="col-lg-3 test" >
```

Comment:
I'm not entirely sure what the `test` class is used for. I cannot find any reference to it anywhere. It can be removed if not used, else I suggest giving it a more descriptive name, ex. `utility-links`.